custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt

```diff
@@ -0,0 +1,87 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import jces1209.vu.MeasureType
+import jces1209.vu.memory.SeededMemory
+import jces1209.vu.page.JiraTips
+import jces1209.vu.page.boards.view.BoardPage
+import jces1209.vu.page.boards.view.ScrumBoardPage
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class WorkOnSprint(
+    private val meter: ActionMeter,
+    private val jiraTips: JiraTips,
+    private val scrumBoardsMemory: SeededMemory<BoardPage>
+) : Action {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    override fun run() {

```

Comment:
This method is too long to fit on a single screen. I'd remove blanklines and extract some private methods to shrink it vertically.